torch/autograd/profiler.py

```diff
@@ -1101,6 +1102,14 @@ def filter_name(name):
     ]
     return name in filtered_out_names
 
+def rewrite_name(name, with_wildcard=False):
```

Comment:
can you leave a comment what this function does? Demangles the name via StringTable?
Also, what is the purpose of wildcard argument?